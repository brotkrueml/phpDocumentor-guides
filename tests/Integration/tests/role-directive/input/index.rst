Some Title
==========

.. role:: custom

An example of using :custom:`interpreted text`

.. role:: custom(emphasis)

:custom:`text`

.. role:: raw-role(raw)
   :format: html latex

:raw-role:`raw text`

.. role:: custom
   :class: special

:custom:`interpreted text`
