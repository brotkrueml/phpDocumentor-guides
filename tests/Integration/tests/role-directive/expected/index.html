<!DOCTYPE html>
<html lang="en">
<head>
    <title>Some Title</title>

</head>
<body>
    <div class="section" id="some-title">
    <h1>Some Title</h1>

            <p>An example of using <span class="custom">interpreted text</span></p>
            <p><em class="custom">text</em></p>
            <p><code class="raw raw-role">raw text</code></p>
            <p><span class="special">interpreted text</span></p>
    </div>

</body>
</html>
